### PR TITLE
vkmusic downloader

### PR DESCRIPTION
**Pull request by hddn**

### CODE REVIEW

File: vkmusic.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+password = getpass.getpass('Enter your password: ')
+session = vk.AuthSession(app_id=APPLICATION_ID,
+                         user_login=login,
+                         user_password=password,
+                         scope='audio')
+api = vk.API(session, v='5.52', lang='en')
+files = api.audio.get()
+files_count = files['count']
+root = os.path.realpath(os.path.dirname(__file__))
+path = os.path.join(root, "downloads")
```
Comment (SpikePyth) [inline vkmusic.py]: use constant

File: vkmusic.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import vk
+
+APPLICATION_ID = '5538517'
+login = input('Enter your login: ')
+password = getpass.getpass('Enter your password: ')
+session = vk.AuthSession(app_id=APPLICATION_ID,
+                         user_login=login,
+                         user_password=password,
+                         scope='audio')
+api = vk.API(session, v='5.52', lang='en')
```
Comment (SpikePyth) [inline vkmusic.py]: :1234: magic numbers

File: vkmusic.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
+from urllib import urlretrieve
+
+import getpass
+import os
+import vk
+
+APPLICATION_ID = '5538517'
```
Comment (SpikePyth) [inline vkmusic.py]: what is `APPLICATION_ID`?

Comment (hddn) [inline vkmusic.py]: id for vk application

Comment (SpikePyth) [inline vkmusic.py]: please provide more specific comment

Comment (hddn) [inline vkmusic.py]: VK_APPLICATION_ID? :)

**Pull request merged**